tests/models/xmod/test_modeling_xmod.py

```diff
@@ -605,3 +605,16 @@ def test_multilingual_batch(self):
         self.assertEqual(output.shape, expected_output_shape)
         # compare the actual values for a slice of last dim
         self.assertTrue(torch.allclose(output[:, :, -1], expected_output_values_last_dim, atol=1e-3))
+
+    @slow
+    def test_fill_mask_pipeline(self):
+        fill_mask_pipeline = pipeline(
+            task="fill-mask",
+            tokenizer="xlm-roberta-base",
+            model="jvamvas/xmod-base",
+            model_kwargs={"default_language": "en_XX"},
+        )
+        src_text = ["Hello, my name is <mask>."]
+        results = [x["token_str"] for x in fill_mask_pipeline(src_text)]
+        expected_results = ["Sarah", "Anna", "Maria", "Amanda", "Jessica"]
+        self.assertListEqual(expected_results, results)
```

Comment:
If it is a pipeline test, let's put it in the `test_pipeline_fill_mask` file. 
Its cool that you added it. I was mentioning something more simpler with juste `model.generate()`